sumdbaudit/audit/service.go

```diff
@@ -62,10 +62,19 @@ func NewService(localDB *Database, sumDB *SumDBClient, height int) *Service {
 	}
 }
 
+// GoldenCheckpoint gets the previously checked Checkpoint, or nil if not found.
+func (s *Service) GoldenCheckpoint(ctx context.Context) *Checkpoint {
+	golden, err := s.localDB.GoldenCheckpoint(s.sumDB.ParseCheckpointNote)
+	if err != nil {
+		return nil
```

Comment:
This seems to silently drop the error, could it return it or at least log it?

Reply:
Logged it for now. I'm eyeing up a refactor of this whole service's API as its a bit clunky at the moment. Ideally it would have something like a `Sync` or `Mirror` method that gets it up to the latest head.